File: common/src/main/java/net/hirafoo/todo/model/Todo.java
package net.hirafoo.todo.model;

import lombok.Data;

@Data
public class Todo {
    public Long id;
    public String name;
    public String description;
    public boolean done;
    // TODO why long?
    public Long term;
    public Long createdAt;
    public Long updatedAt;
}

File: common/src/main/kotlin/net/hirafoo/todo/model/Todo.kt
package net.hirafoo.todo.model

import java.time.LocalDateTime

data class Todo(
    var id: Long = 0,
    var name: String? = null,
    var description: String? = null,
    var done: Boolean = false,
    var term: LocalDateTime = LocalDateTime.MIN,
    //var term: Int = 0,
    //var createdAt: LocalDateTime = LocalDateTime.MIN,
    var createdAt: Int = 0,
    //var updatedAt: LocalDateTime? = null
    var updatedAt: Int = 0
)
